hosts/build02/configuration.nix

```diff
@@ -8,13 +8,14 @@
     ./nixpkgs-update-backup.nix
     inputs.self.nixosModules.common
     inputs.self.nixosModules.hercules-ci
-    inputs.self.nixosModules.raid
     inputs.self.nixosModules.zfs
   ];
 
   # /boot is a mirror raid
   boot.loader.grub.devices = [ "/dev/nvme0n1" "/dev/nvme1n1" ];
   boot.loader.grub.enable = true;
+  boot.swraid.enable = true;
+  boot.loader.grub.extraConfig = "insmod mdraid1x";
```

Comment:
@Lassulus could we load this module in disko when a raid is detected?

Reply:
Experiment in https://github.com/nix-community/disko/pull/329 but this is not meant to block this PR.